cmd/root.go

```diff
@@ -47,19 +46,35 @@ func init() {
 // all other configuration is done by the server with koanf
 // refer to the README.md for more information
 func initConfig() {
-	// Load config from flags, including defaults
 	if err := initCmdFlags(rootCmd); err != nil {
 		log.Fatalf("error loading config: %v", err)
 	}
 
-	c := sentry.LoggerConfig{
-		Debug:  k.Bool("debug"),
-		Pretty: k.Bool("pretty"),
-	}
-
-	logger = c.NewLogger()
+	setupLogging()
 }
 
+// initCmdFlags loads the flags from the command line into the koanf instance
 func initCmdFlags(cmd *cobra.Command) error {
 	return k.Load(posflag.Provider(cmd.Flags(), k.Delim(), k), nil)
 }
+
+func setupLogging() {
+	cfg := zap.NewProductionConfig()
+	if viper.GetBool("pretty") {
+		cfg = zap.NewDevelopmentConfig()
+	}
+
+	if viper.GetBool("debug") {
```

Comment:
This should no longer be viper, and instead the koanf config here and line 63 (and remove the viper import that got added)

```suggestion
	if k.Bool("debug") {
```